README.md

```diff
@@ -1644,6 +1644,7 @@ Software and tools for managing [recipes](https://en.wikipedia.org/wiki/Recipe).
 
 [Remote desktop](https://en.wikipedia.org/wiki/Remote_desktop_software) and [SSH](https://en.wikipedia.org/wiki/Secure_Shell) software for remote management of computer systems.
 
+- [Amnezia VPN](https://en.amnezia.org) - Amnezia is a VPN client with the key feature of deploying your own VPN server on your virtual server. ([Source Code](https://github.com/amnezia-vpn/desktop-client)) `GPL-3.0` `C++/C`
```

Comment:
https://github.com/amnezia-vpn/desktop-client is the source code for the desktop client, but this link should be the source code for the server. I could not find it anywhere after a quick search.

Reply:
Could you explain that? so you need me to change the description or...

Reply:
You need to find the source code for the amnezia server (not the desktop client) and use that as a source code link.

Reply:
Noted.

Reply:
The problem is that users must be able to inspect the source code of what is being deployed to their machine. I couldn't find it after a quick search in the provided source code. So I don't think this should be added in the current state.

Reply:
Alright, I shall close this one as well. Apologies for adding this one.

Reply:
You can find all scripts here - https://github.com/amnezia-vpn/desktop-client/tree/master/client/server_scripts
Also you can check pentest report - https://7asecurity.com/reports/pentest-report-amneziavpn.pdf